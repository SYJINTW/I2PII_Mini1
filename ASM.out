load r0 [0]
load r1 [4]
load r2 [8]
add r3 100 0
add r4 10 0
add r5 r1 0
mul r5 r4 r5
add r5 r3 r5
store [0] r5
store [4] r1
store [8] r2
load r0 [0]
load r1 [4]
load r2 [8]
add r3 r2 0
add r4 100 0
add r5 10 0
mul r5 r4 r5
add r4 50 0
div r4 r5 r4
add r5 10 0
mul r5 r4 r5
add r5 r3 r5
store [4] r5
store [0] r0
store [8] r2
load r0 [0]
load r1 [4]
load r2 [8]
add r3 10 0
add r4 r0 0
mul r4 r3 r4
add r3 100 0
div r3 r4 r3
store [8] r3
store [0] r0
store [4] r1
