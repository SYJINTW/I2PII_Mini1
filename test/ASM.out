load r0 [0]
load r1 [4]
load r2 [8]
add r3 r1 0
add r4 5 0
add r5 r0 0
mul r5 r4 r5
add r5 r3 r5
add r3 2 0
sub r3 r5 r3
add r4 r2 0
add r5 3 0
mul r5 r4 r5
add r5 r3 r5
store [0] r0
store [4] r1
store [8] r2
load r0 [0]
load r1 [4]
load r2 [8]
add r3 5 0
store [0] r3
store [4] r1
store [8] r2
load r0 [0]
load r1 [4]
load r2 [8]
add r3 6 0
store [4] r3
store [0] r0
store [8] r2
load r0 [0]
load r1 [4]
load r2 [8]
add r3 3 0
add r4 5 0
add r4 r3 r4
add r3 8 0
add r6 10 0
add r7 2 0
div r7 r6 r7
mul r7 r3 r7
sub r7 r4 r7
store [0] r7
store [4] r1
store [8] r2
load r0 [0]
load r1 [4]
load r2 [8]
add r3 r0 0
add r4 r0 0
mul r4 r3 r4
add r3 12 0
add r6 12 0
mul r6 r3 r6
sub r6 r4 r6
store [4] r6
store [0] r0
store [8] r2
load r0 [0]
load r1 [4]
load r2 [8]
add r3 r2 0
add r4 r2 0
div r4 r3 r4
add r3 r2 0
mul r6 r3 2
sub r3 r3 r6
add r3 r3 0
mul r6 r3 2
sub r3 r3 r6
add r3 r3 0
mul r6 r3 2
sub r3 r3 r6
add r3 r3 0
mul r6 r3 2
sub r3 r3 r6
add r3 r3 0
add r6 r2 0
sub r6 r3 r6
add r6 r4 r6
add r3 r0 0
add r4 r2 0
mul r4 r3 r4
add r3 r2 0
rem r3 r4 r3
add r3 r6 r3
add r4 r1 0
add r6 r2 0
add r6 r4 r6
add r4 0 0
mul r4 r6 r4
add r4 r3 r4
add r3 r0 0
add r6 r1 0
mul r6 r3 r6
sub r6 r4 r6
store [8] r6
store [0] r0
store [4] r1
load r0 [0]
load r1 [4]
load r2 [8]
add r3 r1 0
mul r4 r3 2
sub r3 r3 r4
add r4 r1 0
mul r6 r4 2
sub r4 r4 r6
mul r4 r3 r4
add r3 r1 0
add r6 r1 0
mul r6 r3 r6
add r3 4 0
add r7 r0 0
mul r7 r3 r7
add r3 r2 0
mul r3 r7 r3
sub r3 r6 r3
sub r3 r4 r3
add r4 2 0
add r6 r0 0
mul r6 r4 r6
add r4 2 0
mul r4 r6 r4
add r6 r0 0
mul r6 r4 r6
div r6 r3 r6
store [0] r6
store [4] r1
store [8] r2
